README.md

```diff
@@ -13,20 +13,20 @@ the following masterclasses:
 
 ## Introduction
 
-Balena applications consist of one or more balena services, which are based upon
-Docker images, containers, volumes and networks. This allows applications to be
+Balena apps consist of one or more balena services, which are based upon
+Docker images, containers, volumes and networks. This allows apps to be
 extremely flexible, allowing services to be isolated with access only to part
 of the host required, as well as defining their own networking and data
 requirements.
 
 This masterclass acts as an introduction to how both single service and
-multicontainer applications operate, as well as demonstrating some of their most
+multicontainer apps operate, as well as demonstrating some of their most
 common uses and some advanced techniques.
 
 In this masterclass, you will learn how to:
 
 * Build a single service
-* Persist data across application updates
+* Persist data across app updates
```

Comment:
```suggestion
* Persist data across fleet updates
```

When things update, we update the fleet or we push a release to the fleet. 
The app gets updated is a result instead.